internal/Picker/Picker.js

```diff
@@ -86,611 +87,249 @@ const allowedClassNames = ['picker', 'valueWrapper', 'joined', 'horizontal', 've
  * @ui
  * @private
  */
+const PickerBase = (props) => {
+	// Set to `true` onFocus and `false` onBlur to prevent setting aria-valuetext (which
+	// will notify the user) when the component does not have focus
+	const [active, setActive] = useState(false);
+	const [pressed, setPressed] = useState(0);
+	const containerRef = useRef();
+	const pickerButtonPressed = useRef(0);
+	const reverseTransition = useRef(false);
+
+	const {
+		'aria-valuetext': ariaValueText,
+		changedBy,
+		children,
+		css: customCss,
+		disabled,
+		id,
+		index,
+		joined,
+		max,
+		min,
+		onChange,
+		onSpotlightDisappear,
+		orientation,
+		reverse,
+		spotlightDisabled,
+		step,
+		value,
+		width,
+		wrap,
+		...rest
+	} = props;
+
+	const css = usePublicClassNames({componentCss, customCss, allowedClassNames});
```

Comment:
The 3rd parameter name should be `publicClassNames` not `allowedClassNames`. This is why the SS tests failed.
```suggestion
	const css = usePublicClassNames({componentCss, customCss, publicClassNames: allowedClassNames});
```